packages/amplication-server/src/core/prismaSchemaParser/prismaSchemaParser.service.ts

```diff
@@ -227,10 +227,35 @@ export class PrismaSchemaParserService {
           continue;
         }
 
-        if (this.isNotAnnotatedRelationField(schema, field)) {
+        if (
+          this.isNotAnnotatedRelationField(schema, field) &&
+          !this.isManyToManyRelation(schema, model, field)
+        ) {
           continue;
         }
 
+        if (this.isManyToManyRelation(schema, model, field)) {
+          const isOneOfTheSideExists = preparedEntities.some((entity) => {
```

Comment:
perhaps name the variable in plural for easier readability (Sides and not Side) - isOneOfTheSidesExists